bitmap-team-awesome/index.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const readBitFile = require('./lib/bit-file-helper');
+const colorTransform = require('./lib/color-transform');
+const EE = require('events');
+const ee = new EE();
+
+ee.on('first', function(){
```

Comment:
@meeksfred While the `first`, `second`, and `third` naming conventions make sense in terms of thier ordering, it's best to name these events something that represents their internal functionality.  For example, your `first` event could be named `grayScale` (or something of that nature) - that way, when another developer works on your code, they will immediately know that `ee.on('grayScale', () => { // do stuff here });` is in relation to that specific change

Additionally, if you were to return `this` at the end of each colorTransform method, you would get the ability to chain your transforms, thus allowing you to have a single readBitFile function that includes something to the degree of:

```
ColorTransform.grayScaleColor(bitmap, bitMapObject)
  .rgbScaleColor(bitmap, bitMapObject)
  .invertColor(bitmap, bitMapObject);
```